### PR TITLE
Add nifcloud client test related to l4lb Part1

### PR DESCRIPTION
**Pull request by sirococoa**
## Summary

- Add tests to the methods related to l4lb in nifcloud_client.go.
  - `DescribeLoadBalancers`
  - `createLoadBalancer`
  - `registerPortWithLoadBalancer`
  - `ConfigureHealthCheck`

- Export nifcloudAPIClient as ExportNifcloudAPIClient for testing `createLoadBalancer` and `registerPortWithLoadBalancer`.
  - d642586da6f7f31b4e01c31e22d1eaff4448c5fa

- Change isAPIError function to a global function.
  - 7a7ddb3f2fd5cc68a8142dde411986d44ec36611

- Corrected acountig type of testdata (`describe_instances.xml` and `describe_instances_not_found_instance_error.xml`) to 1.
  - 1cdca212b7ed2f8856d0730d92c561f471a63845
## Review

- [ ] CI is successful.

### CODE REVIEW

File: pkg/cloudprovider/providers/nifcloud/nifcloud_client_test.go [hunk 1]
```diff
@@ -122,4 +122,243 @@ var _ = Describe("nifcloudAPIClient", func() {
...
+				expectDNSName := "203.0.113.5"
+				testLoadBalancers := helper.NewTestL4LoadBalancer(testLoadBalancerName)
+				gotDNSName, gotErr := nifcloud.ExportCreateLoadBalancer(testNifcloudAPIClient, ctx, &testLoadBalancers[0])
+				Expect(gotErr).ShouldNot(HaveOccurred())
+				Expect(gotDNSName).Should(Equal(expectDNSName))
+			})
+		})
+	})
+
+	Describe("the specified l4 load balancer is already existed", func() {
```
Comment (aokumasan) [inline pkg/cloudprovider/providers/nifcloud/nifcloud_client_test.go:227]: This `Describe` node is outside of  `createLoadBalancer` node.

**Review by aokumasan — CHANGES_REQUESTED** [review]

File: pkg/cloudprovider/providers/nifcloud/nifcloud_client_test.go (hunk 1 above)
Comment (sirococoa) [reply pkg/cloudprovider/providers/nifcloud/nifcloud_client_test.go]: Thank you so much for reviewing. 😸 

I fixed it as 60a83def8bb8423060f30fb8a244679565bb07c9.

Comment (aokumasan) [reply pkg/cloudprovider/providers/nifcloud/nifcloud_client_test.go]: Thanks! 😸

**Review by aokumasan — APPROVED** [review]

**Comment by aokumasan** [comment]
LGTM 🐱

**Pull request merged**